pinot-spi/src/test/java/org/apache/pinot/spi/config/table/IndexingConfigTest.java

```diff
@@ -38,6 +38,12 @@ public void testSerDe()
     IndexingConfig indexingConfig = new IndexingConfig();
     indexingConfig.setLoadMode("MMAP");
     indexingConfig.setAggregateMetrics(true);
+    List<PreAggregationConfig> preAggregationConfigs =
+        Arrays.asList(
+            new PreAggregationConfig("c1", "SUM", "SUM_c1"),
+            new PreAggregationConfig("c2", "MIN", "MIN_c2"),
+            new PreAggregationConfig("c3", "MAX", "MAX_c2"));
```

Comment:
these be double underscore, right? are we missing some validation that this isn't failing?

Reply:
We're allowing the schema and table config to dictate the column name so it can be anything the user wants. In my tests (of the feature), I vary it up.

Reply:
Also, this test is just a serialization/deserialization test so just a string is actually needed.